docker-entrypoint.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-defaultCurrURI=https://raw.githubusercontent.com/Project-Reclass/toynet-django/master
+defaultCurrURI=https://raw.githubusercontent.com/Project-Reclass/toynet-content/tay/data-brainstorm
```

Comment:
should be `master`

Reply:
I think this will actually be called `main` now. GitHub changed their default branch on creation to `main` instead of `master`.

Reply:
wowow